### DISCUSSION TITLE
Welcome to QZQSM Discussions!

### DISCUSSION THREAD

**Discussion by baggio63446333** [baggio63446333/QZQSM#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ellafredeluces** [comment]
Hello, 

Thank you for your QZQSM work. May I know if you have a documentation of the source. For example, purpose of each functions you used? Sorry I am new to coding. Thank you for your help.
